test/basic-querying.test.js

```diff
@@ -469,6 +469,58 @@ describe('basic-querying', function() {
       });
     });
 
+    it('should support "like" that is satisfied', function(done) {
+      User.find({where: {name: {'like': 'John'}}}, function(err, users) {
+        should.not.exist(err);
+        users.should.have.property('length', 1);
```

Comment:
I've updated all per the suggestion. This does not match the rest of the tests in this file - I can update those in another PR.